### DISCUSSION TITLE
Whats Asus monitor control?

### DISCUSSION THREAD

**Discussion by AZTRA081** [seerge/g-helper#3283] [General]
I did a windows clean install recently to 24h2 
And there's 2 new extra processes 
Asus ptp and monitor control.

**Comment by seerge** [comment]
@AZTRA081 hello,

This is what is being installed along Asus Touchpad driver (no idea why, ask Asus).

- Hotplug Control seem to be responsible for hiding that "Eject GPU" tray icon that appears if you enable GPU
- Monitor Control - don't know what it does, but I always just uninstall it personally

**Reply by AZTRA081** [reply]
Oohh okey 
Thanks
I started getting weird frame skipping issues when I switched to 60 hz. And monitor control was something I never saw before so I thought it had something to do with it. That's why I asked 
Thanks again for replying 👍

**Reply by seerge** [reply]
Most likely it's something for external monitors. Like to turn them on(?) when GPU is enabled or so ... but that's just a guess. For me everything works just fine without that.

**Reply by OmerPNW** [reply]
@seerge Do you also recommend uninstalling asus hot plug ? For me, it doesnt even hide the eject gpu tray icon when i switch to charging and g helper switches from eco to standard(GPU detected not sure whats wrong here)

**Reply by OmerPNW** [reply]
And also in ur github page for g helper you mention thta its strongly recommended to run with default windows balance power plan but when i change the modes on g helper they auto change windows power plan(and it does reflect on windows control panel) ,e.g silent -> best power efficiency , balance - > balanced power plan , turbo -> best performance. Do you recommend I stick with these default g helper configs or change all modes to balanced power plan as per the github doc of g helper ? Thank you

**Reply by seerge** [reply]
@OmerPNW hello, it’s up to you :) I don’t know for sure what Hotplug does, but it doesn’t seem to interfere with anything.

As for Power Plan - G-Helper would automatically set this plan in Control Panel if something custom was selected before, mainly yo be as close to default windows behaviour as possible

**Reply by OmerPNW** [reply]
@seerge Got it. Thank you so much for the clarification and response!
